### DISCUSSION TITLE
タスクリストをルーズにするとリストが壊れる

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#309] [要望・アイデア・不具合]
タスクリストはタイトな記述のみをサポートしており、ルーズな記述をサポートしていません。この問題は、ルーズにブロックを含める際に問題になることがあります。

ここで言う「タイト」と「ルーズ」は CommonMark と同等の意味を持ちます。

> A list is [loose](https://spec.commonmark.org/0.30/#loose) if any of its constituent list items are separated by blank lines, or if any of its constituent list items directly contain two block-level elements with a blank line between them. Otherwise a list is [tight](https://spec.commonmark.org/0.30/#tight). (The difference in HTML output is that paragraphs in a loose list are wrapped in `<p>` tags, while paragraphs in a tight list are not.)
>> <https://spec.commonmark.org/0.30/#tight:~:text=A%20list%20is%20loose,tight%20list%20are%20not.)>

![image](https://user-images.githubusercontent.com/75878492/194721172-e2d1927a-1c56-4a92-b547-98bb48462ed2.png)

<details><summary>サンプルコード</summary>

```markdown
- [ ] タスク 1
- [ ] タスク 2

- [ ] タスク 3
- [ ] タスク 4
- [ ] タスク 5
```

</details>

**Comment by kyntk** [comment]
@Yarakashi-Kikohshi ご報告いただきありがとうございます！
不具合だと思われるので、こちらも順次対応します。

**Comment by kyntk** [comment]
@Yarakashi-Kikohshi 
こちらの不具合修正いたしました！
記事を編集していただくとタスクリストとなります！
<img width="852" alt="image" src="https://user-images.githubusercontent.com/17251829/204201532-821af951-5ebb-4dea-96d5-7772d052fb8c.png">

**Reply by yarakos95** [reply]
修正ありがとうございます！
ルーズなタスクリストのチェックボックスがきちんと表示されることを確認しました :eyes:
